tele_giphy/game/giphy.py

```diff
@@ -5,17 +5,57 @@
 KEY = 'dc6zaTOxFJmzC'
 
 
-def gif_random(tag='american psycho', api_key=KEY):
+def _gif_random(tag='american psycho', api_key=KEY, rating=''):
     # See https://github.com/Giphy/GiphyAPI#random-endpoint
     endpoint = GIPHY_URL + 'random'
     params = {'api_key': api_key, 'tag': tag}
+    if not rating:
+        params['rating'] = rating
     data = get(endpoint, params)
     return data
 
 
-def gif_translate(string='leeroy', api_key=KEY):
+def _gif_translate(string='leeroy', api_key=KEY, rating=''):
     # See https://github.com/Giphy/GiphyAPI#translate-endpoint
     endpoint = GIPHY_URL + 'translate'
     params = {'api_key': api_key, 's': string}
+    if not rating:
+        params['rating'] = rating
     data = get(endpoint, params)
     return data
+
+
+# Standarize random and translate endpoint returns
+def giphy_call(call_type='translate', phrase='doge', api_key=KEY, rating=''):
+    standard_data = {'call_type': '',
+                     'image_url': '',
+                     'phrase': '',
+                     'meta': {}}
+    if call_type == 'translate':
+        if not rating:
+            resp = _gif_translate(string=phrase, api_key=api_key)
+        else:
+            resp = _gif_translate(string=phrase, api_key=api_key, rating=rating)
```

Comment:
I meant that the two lines:
```python
resp = _gif_translate(string=phrase, api_key=api_key)
resp = _gif_translate(string=phrase, api_key=api_key, rating=rating)
```
Mean exactly the same thing, because you put the default `rating=''` (in both this function and the _gif...()). So the if statement doesn't do anything.